config-linux.md

```diff
@@ -148,5 +134,60 @@ rootfsPropagation sets the rootfs's mount propagation. Its value is either slave
 
 ## Security
 
-**TODO:** security profiles
+### Linux capabilities
+
+Capabilities is an array that specifies Linux capabilities that can be provided to the process
+inside the container. Valid values are the string after `CAP_` for capabilities defined 
```

Comment:
Probably a good time to migrate these to the new Markdown paragraph format (#82).